docs/self-managed/operational-guides/production-guide.md

```diff
@@ -0,0 +1,530 @@
+        existingSecret: "password-string-literal"
+        audience: "00000000-0000-0000-0000-000000000000"
+        redirectUrl: "https://identity.camunda.example.com"
+        initialClaimName: "email"
+        initialClaimValue: test.user@camunda.com
+      optimize:
+        clientId: "00000000-0000-0000-0000-000000000000"
+        existingSecret: "password-string-literal"
+        audience: "00000000-0000-0000-0000-000000000000"
+        redirectUrl: "https://optimize.camunda.example.com"
+      core:
+        clientId: "00000000-0000-0000-0000-000000000000"
+        existingSecret: "password-string-literal"
+        audience: "00000000-0000-0000-0000-000000000000"
+        tokenScope: "00000000-0000-0000-0000-000000000000/.default"
+        redirectUrl: "https://core.camunda.example.com"
+      console:
+        clientId: "00000000-0000-0000-0000-000000000000"
+        wellKnown: https://login.microsoftonline.com/00000000-0000-0000-0000-000000000000/v2.0/.well-known/openid-configuration
+        audience: "00000000-0000-0000-0000-000000000000"
+        existingSecret: "password-string-literal"
+        redirectUrl: "https://console.camunda.example.com"
+      webModeler:
+        clientId: "00000000-0000-0000-0000-000000000000"
+        clientApiAudience: "00000000-0000-0000-0000-000000000000"
+        publicApiAudience: "00000000-0000-0000-0000-000000000000"
+        redirectUrl: "https://modeler.camunda.example.com"
+core:
+  retention:
+  enabled: true
+  minimumAge: 30d
+  policyName: core-record-retention-policy
+  ingress:
+    grpc:
+      enabled: true
+      className: nginx
+      host: "zeebe-grpc.camunda.example.com"
+      tls:
+        enabled: true
+        secretName: camunda-platform-core-grpc
+identity:
+  externalDatabase:
+    enabled: true
+    host: "external-postgres-host"
+    port: 5432
+    username: "identity_user"
+    database: "identity_db"
+    existingSecret: "identity-db-secret"
+    existingSecretPasswordKey: "database-password"
+webModeler:
+  externalDatabase:
+    url: "jdbc:postgresql://external-postgres-host:5432/camunda_db"
+    user: "camunda_user"
+    password: "secure_password"
+    existingSecret: "camunda-db-secret"
+    existingSecretPasswordKey: "database-password"
+
+elasticsearch:
+  enabled: false
+```
+
+## Other Concepts to take note of
+
+### Upgrade and Maintenance
+
+Make sure secrets are not auto-generated on upgrade.
+
+### Adding more Orchestration clusters
+
+The next recommended step is to setup a multi-namespace deployment. A [guide](/self-managed/setup/guides/multi-namespace-deployment.md) for this is already available. This is the most recommended approach to allow you to setup various environments using the Camunda Orchestration Cluster.
+
+### Running benchmarks
+
+If you would like to run benchmarks on the platform then please refer to our [community project](https://github.com/camunda-community-hub/camunda-8-benchmark)
```

Comment:
Can we consider next steps or call for Action?

What would be the next steps after this guide is completed?

Reply:
I pretty much have mentioned all of these points already. I guess I can rename "Other Concepts to take note of" to "next steps"

that would make things clearer

Reply:
I will resolve this for now. Since no comments have been given yet. If there is still an issue with this section, please ping me here.